### ISSUE TITLE
Requirement is pinned: packaging==19.1

### ISSUE THREAD

**Issue by MartinHjelmare** [fbradyirl/openwrt-luci-rpc#42]
* openwrt-luci-rpc version: 1.1.3
* Python version: Python 3.7
* Operating System: Ubuntu 18.04

### Description

- This library supports an integration in Home Assistant.
- The upcoming pip resolver will be more strict in resolving package version conflicts:
http://pyfound.blogspot.com/2020/03/new-pip-resolver-to-roll-out-this-year.html#what-will-change
- To avoid future issues it would be good if this library would change its pinning to allow any higher version and just set a minimum version.

Thanks for your support of Home Assistant!

### What I Did

```
(home-assistant) x:~/Dev/home-assistant/home-assistant$ pip install -r requirements_all.txt
...
(home-assistant) x:~/Dev/home-assistant/home-assistant$ pipdeptree --reverse --packages packaging
...
packaging==20.4
  - openwrt-luci-rpc==1.1.3 [requires: packaging==19.1]
```

**Comment by onkelbeh** [comment]
Could it be that packaging is only required during build/setup?

**Comment by MartinHjelmare** [comment]
It's currently used to check legacy OpenWRT version:
https://github.com/fbradyirl/openwrt-luci-rpc/blob/8a7ae1a0d67752dec19b85e23fde881ef6440b92/openwrt_luci_rpc/openwrt_luci_rpc.py#L73

**Comment by fbradyirl** [comment]
Fixed in version 1.1.5
